src/common/__tests__/BaseAnnotator-test.ts

```diff
@@ -38,11 +38,19 @@ describe('BaseAnnotator', () => {
         locale: 'en-US',
         token: '1234567890',
     };
-    const getAnnotator = (options = {}): MockAnnotator => new MockAnnotator({ ...defaults, ...options });
-    let annotator = getAnnotator();
+    const annotatorFactory = {
```

Comment:
Is the factory needed or can we continue to use `annotator` directly?

Reply:
Just needed something to house the reference to the current annotator and to ensure we destroy it before creating a new one... open to other ideas